tests/test_grids.h

```diff
@@ -18,6 +18,8 @@
 #include <deal.II/grid/tria_accessor.h>
 #include <deal.II/grid/grid_generator.h>
 
+using namespace dealii;
```

Comment:
That's the wrong fix -- I think the file just needs to `#include <tests/tests.h>`

Reply:
...and that is doing `using namespace dealii;` again. I don't think there is anything else in this header that requires "tests.h".

Reply:
That's right, but at least you do it only once in one central place. All tests have to include tests.h anyway.

Reply:
I don't mind.